sim/scenarios/drop-rate/drop-rate-error-model.h

```diff
@@ -0,0 +1,27 @@
+#ifndef DROPRATE_ERROR_MODEL_H
+#define DROPRATE_ERROR_MODEL_H
+
+#include <set>
+#include <random>
+#include "ns3/error-model.h"
+
+using namespace ns3;
+
+// The DropRateErrorModel drops random packets, at a user-specified drop rate. 
+class DropRateErrorModel : public ErrorModel {
+ public:
+    static TypeId GetTypeId(void);
+    DropRateErrorModel();
+    void SetDropRate(int perc);
+    
+ private:
+    int rate;
+    std::random_device rd;
+    std::mt19937 rng;
```

Comment:
Why are these private members of the class? As far as I can see, you never set them, but just use the value returned by the default constructor. Wouldn't it be sufficient to have them as local variables in the `DropRateErrorModel` constructor?